flink-formats/flink-parquet/src/main/java/org/apache/flink/formats/parquet/avro/AvroParquetRecordFormat.java

```diff
@@ -82,6 +83,7 @@ public Reader<E> createReader(
         return new AvroParquetRecordReader<E>(
                 AvroParquetReader.<E>builder(new ParquetInputFile(stream, fileLen))
                         .withDataModel(getDataModel())
+                        .withConf(HadoopUtils.getHadoopConfiguration(config))
```

Comment:
What happens here if the optional `flink-hadoop-fs` is not on the classpath?

Reply:
It exists in flink runtime https://github.com/apache/flink/blob/master/flink-runtime/pom.xml#L103, so it is not a concern?